src/drivers/driver_zephyr.c

```diff
@@ -50,7 +50,9 @@ void wpa_supplicant_event_wrapper(void *ctx,
 				char *ies = os_zalloc(data->auth.ies_len);
 
 				if (!ies) {
-					wpa_printf(MSG_ERROR, "%s: Failed to alloc ies\n", __func__);
+					wpa_printf(MSG_ERROR,
+					  "%s:%d Failed to alloc ies: %d bytes\n",
```

Comment:
Another small nit. If we want to save flash, we could construct the string so that only one copy is used for all these printouts, like this:
```
wpa_printf(MSG_ERROR,
	   "%s:%d Failed to alloc %s: %d bytes\n",
	   __func__, __LINE__, "ies", data->auth.ies_len);
```

Reply:
We can even drop the `prefix` as we have line number.